client/src/modules/dashboard/Venues/pages/VenuePage.tsx

```diff
@@ -38,7 +38,7 @@ export const VenuePage: NextPageWithLayout = () => {
         </ProgressCardContent>
       </Card>
       <EventList
-        title="Organized By The Venue's Chapter"
+        title="Organized At The Venue"
         events={data.venue.chapter.events}
       />
```

Comment:
What do you think about adding logic to hide `EventList` if `chapter.events.length < 1`?

Reply:
Or maybe add some indication of that in the `EventList` component? What do you think?

Reply:
Good call, `EventList` is used in three pages 👍, and not Venue only.